automatic/msys2/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,46 @@
+﻿$ErrorActionPreference = 'Stop'
+
+$toolsDir = Split-Path $MyInvocation.MyCommand.Definition
+$is64     = (Get-ProcessorBits 64) -and $env:chocolateyForceX86 -ne 'true'
+$dir_name = if ($is64) { 'msys64' } else { 'msys32' }
+
+$pp = Get-PackageParameters
+if (!$pp.InstallDir) { $pp.InstallDir =  "{0}\{1}" -f (Get-ToolsLocation), $dir_name  }
+$install_dir = $pp.InstallDir
+
+if (!(Test-Path $install_dir)) {
+    Write-Host "Installing to:" $install_dir
+    $packageArgs = @{
+        PackageName    = $Env:ChocolateyPackageName
+        FileFullPath   = gi $toolsDir\*-i686*
+        FileFullPath64 = gi $toolsDir\*-x86_64*
+        Destination    = $install_dir
+    }
+    Get-ChocolateyUnzip @packageArgs
+    rm $toolsPath\*.xz -ea 0
+
+    $tarFile = gi "$install_dir\*.tar"
+    Get-ChocolateyUnzip $tarFile $install_dir
+    rm "$install_dir\*.tar" -ea 0
+    $tardir = gi "$install_dir\msys*"
+    mv $tardir\* $install_dir; rm $tardir
+} else { Write-Host "'$install_dir' already exists and will only be updated." }
+
+#https://github.com/msys2/msys2/wiki/MSYS2-installation
+Write-Host "Starting initialization via msys2_shell.cmd"
+Start-Process "$install_dir\msys2_shell.cmd" -Wait -ArgumentList '-c', exit
+
+if (!$pp.NoPath) {  Install-ChocolateyPath $pp.InstallDir }
```

Comment:
The path will need to be removed in an uninstall script.

Reply:
Done